tests/integration/main_test.go

```diff
@@ -60,9 +60,21 @@ func TestMain(m *testing.M) {
 func ConfigureTestEnv(testenv env.Environment) {
 	const receiverMockNamespace = "receiver-mock"
 
+	// List of the namespaces must match values/values_helm_opentelemetry_operator_enabled.yaml
+	// opentelemetry-operator.manager.env.WATCH_NAMESPACE value
+	var openTelemetryOperatorNamespaces = [...]string{"ot-operator1", "ot-operator2"}
+
+	// Helm chart installation extra arguments list
+	helmExtraArgs := []string{"--wait"}
+
 	// Before
 	for _, f := range stepfuncs.IntoTestEnvFuncs(
 		stepfuncs.KubectlApplyFOpt(internal.YamlPathReceiverMock, receiverMockNamespace),
+		// Needed for OpenTelemetry Operator test
+		// TODO: Create namespaces only for specific tests
```

Comment:
Can we create an issue to track this?

Reply:
Sure ;) https://github.com/SumoLogic/sumologic-kubernetes-collection/issues/2199